src/ui/views/Dashboard/index.tsx

```diff
@@ -683,22 +685,32 @@ const Dashboard = () => {
     setTopAnimate('fadeInTop');
   };
   const removeToken = async (removeToken) => {
```

Comment:
Please declare type of function arguments if possible

Reply:
done